test/renate/benchmark/test_scenarios.py

```diff
@@ -49,42 +49,107 @@ def test_failing_to_init(tmpdir, scenario_cls, kwargs):
 
 def test_class_incremental_scenario():
     data_module = DummyTorchVisionDataModule(val_size=0.3, seed=42)
-    class_groupings = [[0, 1, 3], [2], [3, 4]]
+    groupings = ((0, 1, 3), (2,), (3, 4))
     train_data_class_counts = Counter({3: 16, 4: 15, 0: 15, 2: 13, 1: 11})
     val_data_class_counts = Counter({1: 9, 2: 7, 4: 5, 0: 5, 3: 4})
     test_data_class_counts = Counter({0: 20, 1: 20, 2: 20, 3: 20, 4: 20})
-    for i in range(len(class_groupings)):
+    for i in range(len(groupings)):
         scenario = ClassIncrementalScenario(
-            data_module=data_module, class_groupings=class_groupings, chunk_id=i
+            data_module=data_module, groupings=groupings, chunk_id=i
         )
         scenario.prepare_data()
         scenario.setup()
         train_data = scenario.train_data()
         val_data = scenario.val_data()
-        assert len(train_data) == sum([train_data_class_counts[c] for c in class_groupings[i]])
-        assert len(val_data) == sum([val_data_class_counts[c] for c in class_groupings[i]])
+        assert len(train_data) == sum([train_data_class_counts[c] for c in groupings[i]])
+        assert len(val_data) == sum([val_data_class_counts[c] for c in groupings[i]])
         for j, test_data in enumerate(scenario.test_data()):
-            assert len(test_data) == sum([test_data_class_counts[c] for c in class_groupings[j]])
+            assert len(test_data) == sum([test_data_class_counts[c] for c in groupings[j]])
 
 
-def test_class_incremental_scenario_class_grouping_error():
+def test_class_incremental_scenario_groupings_error():
     """Classes selected do not exist in data."""
     scenario = ClassIncrementalScenario(
         data_module=DummyTorchVisionDataModule(val_size=0.3, seed=42),
-        class_groupings=((0, 1, 3), (2, 200)),
+        groupings=((0, 1, 3), (2, 200)),
         chunk_id=0,
     )
     scenario.prepare_data()
     with pytest.raises(ValueError, match=r"Chunk 1 does not contain classes \[200\]."):
         scenario.setup()
 
 
-def test_data_incremental_scenario_init_error():
+def test_data_incremental_scenario_grouping():
+    """Test whether grouping in DataIncrementalScenario works"""
+    scenario = DataIncrementalScenario(
+        data_module=DummyDataIncrementalDataModule(0, (10, 1), val_size=0.2),
+        chunk_id=0,
+        groupings=((0, 1), (2,)),
+    )
+    scenario.prepare_data()
+    scenario.setup()
+    assert set(int(scenario.train_data()[i][1]) for i in range(80)) == {0}
```

Comment:
Can you write a comment about what is being tested for, here? I don't follow the 80, and following 20 are.